README.md

```diff
@@ -73,51 +75,30 @@ How you configure DNS depends on your platform:
+The [detailed documentation](/docs) provides additional information, including an introduction to Envoy and an explanation of how Contour maps key Envoy concepts to Kubernetes.
 
-We've also got [an FAQ][18] for short-answer questions and conceptual stuff that doesn't quite belong in the docs.
+We've also got [an FAQ](/FAQ.md) for short-answer questions and conceptual stuff that doesn't quite belong in the docs.
 
 ## Troubleshooting
 
-If you encounter issues, review the [troubleshooting docs][19], [file an issue][4], or talk to us on the [#contour channel][20] on the Kubernetes Slack server.
+If you encounter issues, review the [troubleshooting docs](/docs/troubleshooting.md), [file an issue][3], or talk to us on the [#contour channel](https://kubernetes.slack.com/messages/contour) on the Kubernetes Slack server.
 
 ## Contributing
 
 Thanks for taking the time to join our community and start contributing!
 
-* Please familiarize yourself with the [Code of Conduct][8] before contributing.
-* See [CONTRIBUTING.md][5] for information about setting up your environment, the workflow that we expect, and instructions on the developer certificate of origin that we require.
-* Check out the [open issues][4].
-* Read how [we're using ZenHub][24] for project and roadmap planning.
+- Please familiarize yourself with the [Code of Conduct](/CODE_OF_CONDUCT.md) before contributing.
+- See [CONTRIBUTING.md](/CONTRIBUTING.md) for information about setting up your environment, the workflow that we expect, and instructions on the developer certificate of origin that we require.
+- Check out the [open issues][3].
 
 ## Changelog
 
-See [the list of releases][6] to find out about feature changes.
-
-[0]: https://github.com/heptio
-[1]: https://travis-ci.org/heptio/contour.svg?branch=master
-[2]: https://travis-ci.org/heptio/contour
-[3]: /docs
-[4]: https://github.com/heptio/contour/issues
-[5]: /CONTRIBUTING.md
-[6]: https://github.com/heptio/contour/releases
-[8]: /CODE_OF_CONDUCT.md
-[9]: https://aws.amazon.com/quickstart/architecture/heptio-kubernetes/
-[10]: /docs/deploy-options.md
-[11]: https://kubernetes.io/docs/concepts/services-networking/service/
-[12]: https://kubernetes.io/docs/concepts/services-networking/ingress/
-[13]: https://www.envoyproxy.io/
-[14]: https://github.com/kubernetes-up-and-running/kuard
-[16]: https://github.com/envoyproxy/envoy/issues/95
-[17]: https://blog.heptio.com/making-it-easy-to-use-envoy-as-a-kubernetes-load-balancer-dde82959f171
-[18]: /FAQ.md
-[19]: /docs/troubleshooting.md
-[20]: https://kubernetes.slack.com/messages/contour
-[21]: https://goreportcard.com/badge/github.com/heptio/contour
-[22]: https://goreportcard.com/report/github.com/heptio/contour
-[23]: /docs/ingressroute.md 
-[24]: /docs/zenhub.md 
+See [the list of releases](https://github.com/heptio/contour/releases) to find out about feature changes.
+
+[1]: /docs/deploy-options.md
```

Comment:
are these three referenced anywhere? If not maybe they can be deleted.

Reply:
Each of these is referenced in more than one place, that's why I kept them as references instead of inline.